content/chainguard/chainguard-images/pricing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Chainguard Container Catalog Pricing"
+linktitle: "Catalog Pricing"
+type: "article"
+description: "Explains Chainguard Container catalog pricing, repository quotas, renaming, and sync, and helps you understand how to make the most of your subscription."
+date: 2025-08-19T08:49:31+00:00
+lastmod: 2025-08-19T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "pricing"]
```

Comment:
I would keep to [approved tags](https://github.com/chainguard-dev/edu/blob/main/TAG_GUIDELINES.md) vs. adding a random one-off here